### PR TITLE
Structures patches

### PR DESCRIPTION
**Pull request by Technicman69**
The `pilgrim_path_temple` revamp process was completed successfully

### CODE REVIEW

File: data/hades/tags/worldgen/biome/any_overworld_biome.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "values": [
```
Comment (Syriusz-2005) [inline data/hades/tags/worldgen/biome/any_overworld_biome.json]: I don't understand those changes.
We need to know if those tags are defining the generation of a pilgrim's path temple

Comment (Syriusz-2005) [reply data/hades/tags/worldgen/biome/any_overworld_biome.json]: Also, I am not sure you added those tag files to the version control system.

Comment (Syriusz-2005) [reply data/hades/tags/worldgen/biome/any_overworld_biome.json]: Yep, I checked and some files are missing, and the naming convention is somewhat incorrect.

Comment (Technicman69) [reply data/hades/tags/worldgen/biome/any_overworld_biome.json]: Whoops, sorry about that. My "search and replace" tool must have disobeyed me and remove those parts of tags names, while I haven't even change those in path.

Comment (Technicman69) [reply data/hades/tags/worldgen/biome/any_overworld_biome.json]: It should be correct now, that I restored them back

**Review by Syriusz-2005 — APPROVED** [review]
I trust you that those changes are fixing all the issues

**Pull request merged**